lib/ansible/plugins/doc_fragments/template_common.py

```diff
@@ -69,6 +69,18 @@ class ModuleDocFragment(object):
     type: str
     default: '}}'
     version_added: '2.4'
+  comment_start_string:
+    description:
+    - The string marking the beginning of a comment statement.
+    type: str
+    default: '{#'
+    version_added: '2.10'
+  comment_end_string:
+    description:
+    - The string marking the end of a comment statement.
+    type: str
+    default: '#}'
```

Comment:
i would not set default this way, since 'real default' will be whatever the engine is currently configured to use

in case you change defaults via a extension loaded in ansible.cfg (or future global settings)

Reply:
@bcoca, you are right, but the other attributes have a default value (such as variable_start_string).